Makefile

```
@@ -85,6 +85,7 @@ debug:
 debug-kill:
 	bash ./test/kill_node.sh
 	pkill -9 -f debug.sh
+	ps aux | grep './test/deploy.sh' | grep -v grep | awk '{print $2}' | xargs kill -9
```

Hey @Frozen, cool zombie process finding, let's kill it with the same approach: 
```suggestion
	pkill -9 -f deploy.sh
```

---

@Frozen can you test if the command works for you on macos ?

---

pkill -9 -f deploy.sh
does not work properly, i don't know why

---

but 
`ps aux | grep './test/deploy.sh' | grep -v grep | awk '{print $2}' | xargs kill -9`
does well

---

Hey @Frozen, can you check than `debug.sh` state via `ps aux | grep './test/debug.sh`?

---

```bash
frozen@Konstantins-Laptop harmony % ps aux | grep deploy
frozen           57897   0.0  0.0 410724112   1488 s011  S+    6:19pm   0:00.01 grep deploy
frozen           57492   0.0  0.0 410773200   4032 s003  S     6:18pm   0:00.03 bash ./test/deploy.sh -B -D 600000 -K 64 -E 64 -L false ./test/configs/local-multi-bls.txt
frozen@Konstantins-Laptop harmony % make debug-kill
bash ./test/kill_node.sh
pkill -9 -f debug.sh
frozen@Konstantins-Laptop harmony % ps aux | grep deploy
frozen           57913   1.2  0.0 410733328   1632 s011  S+    6:19pm   0:00.01 grep deploy
frozen           57492   0.0  0.0 410773200   4096 s003  S     6:18pm   0:00.03 bash ./test/deploy.sh -B -D 600000 -K 64 -E 64 -L false ./test/configs/local-multi-bls.txt
frozen@Konstantins-Laptop harmony %
```

i got the reason, `pkill -9 -f debug.sh` kills debug, but i need to kill deploy.sh as well

---

so, `pkill -9 -f deploy.sh` does well
```bash
frozen@Konstantins-Laptop harmony % ps aux | grep deploy
frozen           57925   0.0  0.0 410743568   1792 s011  S+    6:22pm   0:00.01 grep deploy
frozen           57492   0.0  0.0 410773200   3728 s003  S     6:18pm   0:00.03 bash ./test/deploy.sh -B -D 600000 -K 64 -E 64 -L false ./test/configs/local-multi-bls.txt
frozen@Konstantins-Laptop harmony %
frozen@Konstantins-Laptop harmony %
frozen@Konstantins-Laptop harmony % pkill -9 -f deploy.sh
frozen@Konstantins-Laptop harmony % ps aux | grep deploy
frozen           57928   0.0  0.0 410733328   1632 s011  S+    6:22pm   0:00.01 grep deploy
frozen@Konstantins-Laptop harmony %
```

---

Nice lets use pkill in both places :pray: